ansible/roles/fews/files/upload_to_google.sh

```diff
@@ -1,20 +1,23 @@
 #!/bin/bash
 # settings
 # local directory with the data to upload
-export localfolder="/srv/mnt/glossis_test/export/Google"
-export localfolder_gloffis="/srv/mnt/gloffis/latest/W3RA_05deg/ECMWF-CF"
-export KUBECONFIG=~/.kube/kube.conf
-export GOOGLE_APPLICATION_CREDENTIALS=~/google-credentials.json
+export localfolder="/opt/fews/datacollect/fromfss/Export/Google"
+export localfolder_gloffis="/opt/fews/datacollect/tofss/Import/GLOFFIS"
+export KUBECONFIG=/home/fews/kube.conf
+export GOOGLE_APPLICATION_CREDENTIALS=/home/fews/google-credentials.json
+export BOTO_CONFIG=/home/fews/boto
+
+cd /opt/fews/datacollect/software/Upload
 
 # Remove all old files from folder
 find ${localfolder} -type f -name '*.nc' -mmin +60 -exec rm {} \;
```

Comment:
Is this removal time limit still relevant?

Reply:
yes it is, otherwise we end up we 2, 3 4 sets of 16 netcdfs. The code can nog handle that at the moment so we only have the latest model output available in the bucket